go.mod

```
@@ -111,4 +111,12 @@ replace (
 	k8s.io/kubernetes => k8s.io/kubernetes v1.14.1
 )
 
-replace github.com/operator-framework/operator-lifecycle-manager => github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190605231540-b8a4faf68e36
+replace (
+	// Indirect operator-sdk dependencies use git.apache.org, which is frequently
+	// down. The github mirror should be used instead.
+	// go.opencensus.io@v0.18.0
+	git.apache.org/thrift.git v0.0.0-20180902110319-2566ecd5d999 => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

```suggestion
	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```
Is this replacement only for a specific version?
Same for Line 120.

---

Yeah. Pinning all dependencies using `git.apache.org/thrift` to the same `github.com/apache/thrift` version may break a dependency, and even though it doesn't appear to be the case here, I wanted to be safe. I'm ok with applying your suggested change though.

---

If using a single version doesn't break the dependency I would say we use that.

The issue I see with this (and correct me if I'm wrong), is if the indirect dependency changes to use some other version then the replace command will no longer swap out `git.apache.org/thrift.git => github.com/apache/thrift` for that version.

And since we use the proxy in our CI we won't know if it breaks again.